StringFormatterBench/TraceEvent.ReadMe.txt

```diff
@@ -0,0 +1,79 @@
+﻿
+************* Welcome to the  Microsoft.Diagnostics.Tracing.TraceEvent library!  ***************
```

Comment:
I'm not too keen about it either, that's just what happens from adding the NuGet for the memory diagnosis, I'll follow your lead and remove it.